### DISCUSSION TITLE
Wezterm not showing image in neofetch

### DISCUSSION THREAD

**Discussion by mobsenpai** [wezterm/wezterm#4431] [Q&A]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

AwesoemWM

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

I am using this command 
`neofetch --iterm2 ~/image.png`
but wezterm is not showing the image at all. It worked in other terminal fyi.

**Comment by bew** [comment]
Can you share the image that doesn't work?

**Reply by mobsenpai** [reply]
Well no image is working for me still. https://imgur.com/a/nZDrqGe
One more thing is that I am using nixos and have installed it from home manager if that helps.
And I am using fresh neofetch install without any config edit.

**Comment by mobsenpai** [comment]
@wez Can you look into it?

**Comment by mobsenpai** [comment]
Giving up

**Comment by mobsenpai** [comment]
For anyone wondering installing imagemagick did the trick
